src/javaScript/indexDom.js

```diff
@@ -47,3 +47,32 @@ document.addEventListener('DOMContentLoaded', () => {
 
   updatePage()
 })
+
+fetch('src/javaScript/games.json')
+  .then((response) => {
+    if (!response.ok) {
+      throw new Error('Network response was not ok')
+    }
+    return response.json()
+  })
+  .then((data) => {
+    const allGames = data.payload.gamesDetail
+    allGames.forEach((game) => {
+      const gamesContainer = document.querySelector('.grid-index')
```

Comment:
Do you think we need to query the selector in a loop? Or is it the same element so we can query it only once?